stable/gocd/values.yaml

```diff
@@ -70,6 +70,9 @@ server:
   #     cpu: 100m
   #     memory: 1024Mi
 
+  # Sidecar containers that runs alongside GoCD server.
```

Comment:
@igno Can you add a bit more documentation here. Something like the documentation near initContainers below?

https://github.com/helm/charts/blob/afd78ccd8056213eb06aeceabbd651c87b4ad88b/stable/gocd/values.yaml#L78-L88

Reply:
Added a small example with a volumeMount (as that's typically the use case for a sidecar).